site/content/docs/5.1/examples/cheatsheet/index.html

```diff
@@ -1156,13 +1156,13 @@ <h3>Navs</h3>
           </nav>
           <div class="tab-content" id="nav-tabContent">
             <div class="tab-pane fade show active" id="nav-home" role="tabpanel" aria-labelledby="nav-home-tab">
-              <p><strong>This is some placeholder content the Home tab's associated content.</strong> Clicking another tab will toggle the visibility of this one for the next. The tab JavaScript swaps classes to control the content visibility and styling. You can use it with tabs, pills, and any other <code>.nav</code>-powered navigation.</p>
```

Comment:
[Bootstrap] Create a PR to make the same modification. Forgot to do it after the modification in nav tabs.

Reply:
See https://github.com/twbs/bootstrap/pull/36196